src/components/Navbar/Navbar.css

```diff
@@ -1,5 +1,11 @@
 .header-wrapper {
   box-shadow: 0px 0px 12px -6px var(--BLACK);
+  position: sticky;
+  background-color: white;
+  z-index: 2;
+  top: 0;
```

Comment:
Suggestion: Add z-index in variable so that it would make easy to manage all the z-indexes in one place.

example:
   ```
   :root {
    --z-index-1: 1;
    --z-index-2: 2;
    --z-index-3: 3;
    --z-index-4: 4;
    --z-index-5: 5;
    }
```